### PR TITLE
add function

### PR DESCRIPTION
**Pull request by YifanLi9**

### CODE REVIEW

File: math/FunMath.cpp [hunk 1]
```diff
@@ -1 +1,10 @@
 #include "FunMath.h"
+#include <vector>
+
+
+std::vector<int> squaredvector(int squared) {
```
Comment (muzny) [inline math/FunMath.cpp]: - Add a function comment: what does it do, what are the parameters/return values
   - Make sure to follow our style guide for functions NamedLikeThis

File: math/FunMath.cpp [hunk 2]
```diff
@@ -1 +1,10 @@
...
+  std::vector<int> return_vector;
+  for (int i = 1 ; i <= squared ; i++ )
```
Comment (muzny) [inline math/FunMath.cpp]: Use { } braces to avoid bugs!

File: math/FunMath.h [hunk 3]
```diff
@@ -17,6 +17,9 @@
     @return the sum of the two numbers
 */
 // int Sum(int num1, int num2);
+<<<<<<< HEAD
```
Comment (muzny) [inline math/FunMath.h]: don't leave merge markings in!

File: math/FunMath.h [hunk 4]
```diff
@@ -17,6 +17,9 @@
...
+std::vector<int>  squaredvector(int squared);
+=======
```
Comment (muzny) [inline math/FunMath.h]: also here

File: math/FunMath.h [hunk 5]
```diff
@@ -25,5 +28,6 @@ std::vector<int> multiples(int n, int m);
 std::string RemoveSubstring(std::string s1, std::string s2);
 
 std::vector<int> multiples(int n, int m);
+>>>>>>> master
```
Comment (muzny) [inline math/FunMath.h]: and here

**Comment by jatr6994** [comment]
I believe have I have rally messed up..... I merged pjc into master

File: math/FunMath.cpp (hunk 1 above)
Comment (yaya7263) [reply math/FunMath.cpp]: Added comment

File: math/FunMath.cpp (hunk 2 above)
Comment (yaya7263) [reply math/FunMath.cpp]: done

**Pull request closed without merging**